main-command/src/main/scala/sbt/BasicCommandStrings.scala

```diff
@@ -96,6 +97,14 @@ $HelpCommand <regular expression>
 	The order is preserved between all early commands, so `sbt "early(a)" "early(b)"` executes `a` and `b` in order.
 """
 
+  def addPluginSbtFileHelp = {
+    val brief =
+      (s"--$AddPluginSbtFileCommand=<file>", "Adds the given *.sbt file to the plugin build.")
```

Comment:
aren't most options with a single dash (at least in the help)?

Reply:
The single line convention comes from Java, and since Java 9 is updating to more UNIX `--` style, I am following the cargo cult.